default.nix

```diff
@@ -25,10 +25,11 @@ let
       }
     else if info.type == "git" then
       { outPath =
-          builtins.fetchGit
-            ({ url = info.url; }
+          builtins.fetchTree
```

Comment:
We shouldn't use `fetchTree` because it's an experimental feature and is not available in older versions of Nix (so using it would kind of negate the point of `flake-compat`).